packages/react-core/src/demos/Skeleton.md

```diff
@@ -0,0 +1,12 @@
+---
+id: Skeleton
+section: components
+---
+
```

Comment:
I believe you'll want to add the `import { DashboardWrapper } from '@patternfly/react-core/src/demos/DashboardWrapper';` import statement to the top of this markdown file and that _should_ fix the docs build